bindings/ios/MEGARequest.mm

```diff
@@ -187,7 +187,7 @@ - (MEGAFolderInfo *)megaFolderInfo {
 }
 
 - (NSDictionary<NSString *, MEGAStringList *> *)megaStringListDictionary {
-    MegaStringListMap *map = self.megaRequest->getMegaStringListMap();
+    MegaStringListMap *map = self.megaRequest->getMegaStringListMap()->copy();
```

Comment:
Not sure if you really need to make a copy here. The `map` is not deleted here, not sure if that would represent a memory-leak?

Reply:
I guess what you want to do is to keep valid `MegaStringList` objects, which should be done by `copy()` on the items of the map, not copying the full map and keeping it alive forever.

Reply:
Yes, after looking at this more closely the `keyList` should be deleted and key and value should be copied into `dict`. Then a call to `copy` isn't necessary. I will talk to Simon about this.

Reply:
Thanks for point out this issue. I have updated the iOS binding code according to Christian's suggestions.